apps/studio/src/store/modules/TabModule.ts

```diff
@@ -53,6 +55,16 @@ export const TabModule: Module<State, RootState> = {
     },
     setActive(state, tab?: OpenTab) {
       state.active = tab
+    },
+    addLastClosedTab(state, tabs: OpenTab[]){
```

Comment:
don't think you need this, it can just be done in the `remove` mutation.

so remove would do:

```js
    remove(state, tab: OpenTab) {
      state.tabs = _.without(state.tabs, tab)
      state.lastClosedTabs.push(tab)
    },
```